README.md

```diff
@@ -126,60 +119,26 @@ See [arguments explanations here](#arguments).
   just create -t <language> -d <destination> -- -w -l
   ```
 
-## Containerized
-
-TODO: Add this.
-
 # Structure
 
 The following describes the content of the top-level directories:
 
-- [`docs`](src/generic/docs) : The top-level folder to any related
-  documentation. The [README.md](src/generic/README.md) should link into this
+- [`docs`](src/generic/docs) : All documentation-related files. The [README.md](src/generic/README.md) should link into this
   folder.
-- [`examples`](src/generic/examples) : The top-level folder which should contain
-  some examples how to use this software component.
-- [`external`](src/generic/external) : If you really want to use sub-modules
-  (which you generally should avoid for a multitude of reasons), your external
-  stuff should be either placed in here or in `src/external` if its more related
-  to your source.
-- `src`: The folder where your source code lives.
-- [`tools`](src/generic/tools): The folder for all specific needs:
-  - [`configs`](src/generic/tools/configs): A collection folder for all config
-    related files for certain tools like, e.g. formatters, linters etc.
-  - [`nix`](src/generic/tools/nix): The folder containing all Nix related stuff.
-  - [`ci`](src/generic/tools/ci): Folder containing all CI related
-    tooling/scripts.
-  - [`scripts`](src/generic/tools/scripts): Folder containing additional scripts
-    complementing the `justfile` etc.
+- [`examples`](src/generic/examples) : Examples showing how to use this software component.
+- [`external`](src/generic/external) : Third party resources imported with git submodules, [vendir](https://carvel.dev/vendir/) or other tools.
+- `src`: Where your source code lives.
+- [`tools`](src/generic/tools): Specific needs which are not part of the source:
+  - [`configs`](src/generic/tools/configs): config related files for certain tools like, e.g. formatters, linters etc.
+  - [`nix`](src/generic/tools/nix): Nix related stuff.
+  - [`ci`](src/generic/tools/ci): CI related tooling/scripts.
+  - [`scripts`](src/generic/tools/scripts): Additional scripts complementing the `justfile` etc.
 
 ## Generic Template
 
 - [Template Source](src/generic)
 - [Demo Rendering](https://github.com/sdsc-ordes/repository-template-generic)
 
-### Features
```

Comment:
question: @cmdoret  was that intentional to be removed?

Reply:
Yeah because that's basically a repeat of the feature list in the intro.